Assets/Tests/InputSystem.Editor/InputActionsEditorTests.cs

```diff
@@ -0,0 +1,261 @@
+// UITK TreeView is not supported in earlier versions
+// Therefore the UITK version of the InputActionAsset Editor is not available on earlier Editor versions either.
+#if UNITY_EDITOR && UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS && UNITY_6000_0_OR_NEWER
+
+using NUnit.Framework;
+using System;
+using System.Collections;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.Editor;
+using UnityEngine.TestTools;
+using UnityEngine.UIElements;
+
+public class InputActionsEditorTests
+{
+    #region setup and teardown
+    InputActionsEditorWindow m_Window;
+    InputActionAsset m_Asset;
+
+    [OneTimeSetUp]
+    public void OneTimeSetUp()
+    {
+        TestUtils.MockDialogs();
+        m_Asset = AssetDatabaseUtils.CreateAsset<InputActionAsset>();
+        m_Asset.AddActionMap("First Name");
+        m_Asset.AddActionMap("Second Name");
+        m_Asset.AddActionMap("Third Name");
+    }
+
+    [OneTimeTearDown]
+    public void OneTimeTearDown()
+    {
+        AssetDatabaseUtils.Restore();
+        TestUtils.RestoreDialogs();
+    }
+
+    [UnitySetUp]
+    public IEnumerator UnitySetup()
+    {
+        var editor = InputActionsEditorWindow.OpenEditor(m_Asset);
+        m_Window = editor;
+        yield return WaitForNotDirty(editor.rootVisualElement);
+    }
+
+    [TearDown]
+    public void TearDown()
+    {
+        m_Window?.Close();
+    }
+
+    #endregion
+
+    #region Helper methods
+
+    void Click(VisualElement ve)
```

Comment:
thanks I will make some name adjustment.
Most of the methods are copied from https://internaldocs.unity.com/editor_and_runtime_development_guide/TestAutomation/TestFrameworks/UITestFramework/CodeSamples/code-samples/ and will be replaced by the helpers on neutron

Reply:
For the ve => target is unfortunately not as simple. For clicking it may work because it took the positon and if the target it on top the click will be dispatched to it.
For all other command even if you call visualelement.sendevent it will be dispatched to the panel which will search for te focused element.
I make some change to make it clearer